src/EA.Weee.Web/ViewModels/OrganisationRegistration/Type/TonnageTypeViewModel.cs

```diff
@@ -0,0 +1,29 @@
+﻿namespace EA.Weee.Web.ViewModels.OrganisationRegistration.Type
+{
+    using Core.Organisations;
+    using Shared;
+    using System.ComponentModel.DataAnnotations;
+
+    public class TonnageTypeViewModel : RadioButtonStringCollectionViewModel
```

Comment:
@will-hope98 We could add unit tests to test the required attribute on the model but as this is quite a simple model no real need. 

for info an example test to check for attributes that has been done before is...

`        [Theory]
        [InlineData("ReferenceDisplay", "Reference ID")]
        [InlineData("StartDate", "Start date")]
        [InlineData("EndDate", "End date")]
        [InlineData("WasteTypeValue", "Obligation type")]
        [InlineData("ProtocolValue", "Actual or protocol")]
        [InlineData("SubmittedDate", "Submitted date")]
        [InlineData("RejectedDate", "Date rejected")]
        [InlineData("ReturnedDate", "Date returned")]
        [InlineData("ApprovedDate", "Date approved")]
        [InlineData("ReturnedReason", "Reason")]
        [InlineData("RejectedReason", "Reason")]
        public void EvidenceNoteViewModel_Properties_ShouldHaveDisplayAttributes(string property, string display)
        {
            typeof(EvidenceNoteViewModel).GetProperty(property).Should()
                .BeDecoratedWith<DisplayNameAttribute>(a => a.DisplayName.Equals(display));
        }`